lib/spl/include/spl.h

```diff
@@ -357,60 +381,209 @@ typedef struct UnkSPLStruct1_t {
     } unk_38;
     s32 unk_3C;
     struct {
-        UnkSPLStruct6 * unk_00;
+        SPLEmitter * unk_00;
         const MtxFx43 * unk_04;
     } unk_40;
     u16 unk_48;
     u16 unk_4A;
-} UnkSPLStruct1; // size=0x4c
-
-void SPL_0209C400(UnkSPLStruct1 * param0);
-void SPL_0209C444(UnkSPLStruct1 * param0, UnkSPLStruct6 * param1);
-UnkSPLStruct6 * SPL_0209C4D8(UnkSPLStruct1 * param0, int param1, void (* param2)(struct UnkSPLStruct6_t *));
-void SPL_0209C5E0(UnkSPLStruct1 * param0, const MtxFx43 * param1);
-UnkSPLStruct6 * SPL_0209C56C(UnkSPLStruct1 * param0, int param1, const VecFx32 * param2);
-void SPL_0209C6A8(UnkSPLStruct1 * param0);
-BOOL SPL_0209C7E0(UnkSPLStruct1 * param0);
-BOOL SPL_0209C7F4(UnkSPLStruct1 * param0);
-BOOL SPL_0209C808(UnkSPLStruct1 * param0, u32 (* param1)(u32, BOOL));
-BOOL SPL_0209C8BC(UnkSPLStruct1 * param0, u32 (* param1)(u32, BOOL));
-void SPL_0209C988(UnkSPLStruct1 * param0, const void * param1);
-UnkSPLStruct1 * SPL_0209CD00(UnkFuncPtr_0209CD00 param0, u16 param1, u16 param2, u16 param3, u16 param4, u16 param5);
-
-void SPL_020A1E30(const void * param0, UnkSPLStruct8 * param1, VecFx32 * param2, struct UnkSPLStruct6_t * param3);
-void SPL_020A1EC4(const void * param0, UnkSPLStruct8 * param1, VecFx32 * param2, struct UnkSPLStruct6_t * param3);
-void SPL_020A1FE0(const void * param0, UnkSPLStruct8 * param1, VecFx32 * param2, struct UnkSPLStruct6_t * param3);
-void SPL_020A20B8(const void * param0, UnkSPLStruct8 * param1, VecFx32 * param2, struct UnkSPLStruct6_t * param3);
-void SPL_020A213C(const void * param0, UnkSPLStruct8 * param1, VecFx32 * param2, struct UnkSPLStruct6_t * param3);
-void SPL_020A2204(const void * param0, UnkSPLStruct8 * param1, VecFx32 * param2, struct UnkSPLStruct6_t * param3);
-
-static inline void SPL_UnkInline1 (UnkSPLStruct6 * param0, const VecFx32 * param1)
+} SPLManager; // size=0x4c
+
+typedef struct SPLGravity {
+    VecFx16 mag;
+    u16 reserved0;
+} SPLGravity;
+
+typedef struct SPLRandom {
+    VecFx16 mag;
+    u16 intvl;
+} SPLRandom;
+
+typedef struct SPLMagnet {
+    VecFx32 pos;
+    fx16 mag;
+    u16 reserved0;
+} SPLMagnet;
+
+typedef struct SPLSpin {
+    u16 radian;
+    u16 axis_type;
+} SPLSpin;
+
+typedef struct SPLSimpleCollisionField {
+    fx32 y;
+    fx16 coeff_bounce;
+    
+    struct {
+        u16 eventtype : 2;
+        u16 global : 1;
+        u16 reserved : 13;
+    } etc;
+} SPLSimpleCollisionField;
+
+typedef struct SPLConvergence {
+    VecFx32 pos;
+    fx16 ratio;
+    u16 reserved0;
+} SPLConvergence;
+
+typedef struct SPLNode {
```

Comment:
This is just an alias for an SPLEmitter

Reply:
No it isn't. It's used for `SPLParticle` as well.